tests/unit/s2n_kem_test.c

```diff
@@ -121,54 +122,6 @@ int main(int argc, char **argv)
         EXPECT_EQUAL(sizeof(kem_shared_secret_size), 2);
         EXPECT_EQUAL(sizeof(kem_ciphertext_key_size), 2);
     };
-    {
-        struct s2n_kem_params server_kem_params = { 0 };
-        server_kem_params.kem = &s2n_test_kem;
```

Comment:
Are these test cases covered elsewhere, and that's why they're safe to remove?

Reply:
I did not remove any test cases, only moved the tests to be further down in the file. This was so that we are not unnecessarily running the same tests twice for tests that don't depend on the value of `len_prefixed` when I added the for loop that tested both values.

First, we run all tests that don't depend on the value of `len_prefixed`, as indicated by the comment on line 116. Then we run all other tests in a for loop that tests with both possible values of `len_prefixed` later on in line 196.